jekyll/_cci2/orb-author.md

```diff
@@ -9,279 +9,289 @@ version:
+You will also be asked for the remote git repository URL which we obtained in step 1. Ensure you are using the `HTTPS` URL.
 
-When developing your own orb, you may find it useful to write an inline orb. The section below describes how you can write your own inline orb.
+#### 5) When prompted, in a separate terminal, push the changes up to Github.
+{:.no_toc}
 
-### Writing Inline Orbs
+During the setup process, the `orb init` command will take several steps to prepare your automated orb development pipeline. The modified template code produced by the CLI must be pushed to the repository before the CLI can continue and automatically follow your project on CircleCI.com successfully.
 
-Inline orbs can be handy during development of an orb or as a convenience for name-spacing jobs and commands in lengthy configurations, particularly if you later intend to share the orb with others.
+```bash
+$ git push origin master
+```
 
-To write inline orbs, place the orb elements under that orb's key in the `orbs` declaration in the configuration. For example, if you want to import one orb and then author inline for another, the orb might look like the example shown below:
+For technical reasons, we can not automatically push this code.
+Once complete, return to the CLI window and confirm the changes have been pushed.
 
-{% raw %}
-```yaml
-version: 2.1
-description: # The purpose of this orb
-
-orbs:
-  my-orb:
-    orbs:
-      codecov: circleci/codecov-clojure@0.0.4
-    executors:
-      specialthingsexecutor:
-        docker:
-          - image: circleci/ruby:2.7.0
-    commands:
-      dospecialthings:
-        steps:
-          - run: echo "We will now do special things"
-    jobs:
-      myjob:
-        executor: specialthingsexecutor
-        steps:
-          - dospecialthings
-          - codecov/upload:
-              path: ~/tmp/results.xml
-
-workflows:
-  main:
-    jobs:
-      - my-orb/myjob
+#### 6) The CLI will finish by automatically following the project on CircleCI and generating the first development version fo your orb for testing (a hello-world sample).
+{:.no_toc}
+
+You are now ready to push changes to your orb and automatically publish your orb. We'll go over deploying changes with semver versioning in the [Orb Publishing Process]({{site.baseurl}}/2.0/creating-orbs) docs.
+
+### Writing Your Orb
+
+Before you begin working on your orb, ensure you are on a non-default branch. We typically recommend starting your orb on the `alpha` branch.
+
+```shell
+$ git checkout -b alpha
 ```
-{% endraw %}
 
-In the example above, note that the contents of `my-orb` are resolved as an inline orb because the contents of `my-orb` are a map; whereas the contents of `codecov` are a scalar value, and thus assumed to be an orb URI.
+Once you have ran the `circleci orb init` command, you will have a repository with importantly, a `.circleci` and `src` directory.
```

Comment:
Does this mean -  run `circleci orb init` now? Or would that already have happened in the steps above?

Reply:
Oh I see it's step 2 above